.gitignore

```diff
@@ -7,4 +7,5 @@ debug
 bin/
 *.env
 etc/
-root/
\ No newline at end of file
+root/
+external/
```

Comment:
Out of interest, what's this created by?

Reply:
I maintain external files, like .cue, that I want ignored

Reply:
I'll remove it, as I hadn't planned for this PR to be reviewed until I changed its status to open 😊